### ISSUE TITLE
Updates log

### ISSUE THREAD

**Issue by YubinXie** [matterport/Mask_RCNN#461]
Hi Team,
Thanks for the great codes. I noticed that there is a recent update. It would be nice that if you can write something about what you update/change in every updates. Thanks!

**Comment by waleedka** [comment]
Yes, good point. I'm planning to make a new release in the next week or so, and I'll make sure to include a summary of what changed since the earlier release. In the meantime, I do try to put good comments in every (or most) commits. 

https://github.com/matterport/Mask_RCNN/commits/master

**Comment by YubinXie** [comment]
Sounds good. One more thing I noticed is that, new release sometimes cause new bugs. Knowing what is changed will help us debug! 

BTW, Thank you for your great work.
